src/resource.rc

```diff
@@ -128,7 +128,7 @@ STYLE 0
 CAPTION "About"
 FONT 8, "MS Shell Dlg"
 BEGIN
-	LTEXT "EarthBound Music Editor v2.4.0\r\nReleased August 27, 2021\r\nProgrammed by Goplat\r\nAdditional programming by BlueStone, phoenixbound, and vince94\r\n", 0,
+	LTEXT "EarthBound Music Editor v2.4.0\r\nReleased August 27, 2021\r\nProgrammed by Goplat\r\nAdditional programming by BlueStone, phoenixbound, and livvy94\r\n", 0,
```

Comment:
There are too many contributors at this point with some not even listed here. I'm going to be replacing this with a link to the github repo when I update the version to 2.5.0 for the next release.